drivers/scheduler/rke/rke.go

```diff
@@ -67,6 +68,7 @@ func (r *Rancher) GetRancherClusterParametersValue() (*RancherClusterParameters,
 	var rkeToken string
 	// TODO Rancher URL for cloud cluster will not be fetched from master node IP
 	masterNodeName := node.GetMasterNodes()[0].Name
+	log.Infof(" The master node here is %v", masterNodeName)
```

Comment:
nit: remove the space after double quotes

Reply:
done